packages/wifi_iot/android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -93,6 +93,11 @@ public class WifiIotPlugin
   private static final Map<Integer, Result> resultMap = new HashMap<Integer, Result>();
   private static final int ACTIVITY_RESULT_REQUEST_CODE_ADD_NETWORKS = 66778899;
 
+  // Register network reuslts
```

Comment:
fix spelling and wording; for example, "register network result flags"

Reply:
Fixed